austrakka/components/metadata/funcs.py

```diff
@@ -19,37 +24,81 @@
 
 @logger_wraps()
 def add_metadata(
-    file: BufferedReader,
-    proforma_abbrev: str,
+        file: BufferedReader,
+        proforma_abbrev: str,
         blanks_will_delete: bool,
+        batch_size: int,
 ):
     path = "/".join([SUBMISSION_PATH, SUBMISSION_UPLOAD])
     if blanks_will_delete:
         path = f"{path}?{DELETE_ON_BLANK_PARAM}"
-    _call_submission(path, file, proforma_abbrev)
+    _call_batched_submission(path, file, proforma_abbrev, batch_size)
 
 
 @logger_wraps()
 def append_metadata(
-    file: BufferedReader,
-    proforma_abbrev: str,
+        file: BufferedReader,
+        proforma_abbrev: str,
         blanks_will_delete: bool,
+        batch_size: int,
 ):
     path = "/".join([SUBMISSION_PATH, SUBMISSION_UPLOAD_APPEND])
     if blanks_will_delete:
         path = f"{path}&{DELETE_ON_BLANK_PARAM}"
-    _call_submission(path, file, proforma_abbrev)
+    _call_batched_submission(path, file, proforma_abbrev, batch_size)
```

Comment:
Why rename _call_submission? The batch_size parameter already indicates that batching should be expected.

Reply:
I am jumping the gun. Must be the coffee. You're using both!